core/src/avm2/globals/flash/display/displayobject.rs

```diff
@@ -907,7 +997,12 @@ pub fn create_class<'gc>(mc: MutationContext<'gc, '_>) -> GcCell<'gc, Class<'gc>
         ("scaleX", Some(scale_x), Some(set_scale_x)),
         ("x", Some(x), Some(set_x)),
         ("y", Some(y), Some(set_y)),
+        ("z", Some(z), Some(set_z)),
```

Comment:
please add a TODO comment that this needs to be version-checked (as this will breaks SWF9 if they used `z` without `override`)